_episodes/07-visualization-ggplot-python.md

```diff
@@ -293,7 +293,7 @@ group, a boxplot can be used:
 
 ![png](../fig/06_boxplot.png)
```

Comment:
I know you haven't touched this in this PR, but anyway - if you can come up with a better title for the figure - that would be great

Reply:
How about changing `png` to:
> A box plot of species' weight versus their id.

Reply:
I'm fine with this, I'd just rather do it in a separate pull request.  I'm going to mark this as resolved for now.